bigquery_etl/cli/stage.py

```diff
@@ -286,7 +294,11 @@ def _update_references(artifact_files, project_id, dataset_suffix, sql_dir):
         name_pattern = name.replace("*", r"\*")  # match literal *
         original_dataset = artifact_file.parent.parent.name
         deployed_dataset = original_dataset
-        if dataset_suffix:
+        if (
+            dataset_suffix
+            and "region-us" not in original_dataset
+            and "INFORMATION_SCHEMA" not in original_dataset
+        ):
```

Comment:
only append data suffix for datasets that are not `region-us` or `INFORMATION_SCHEMA`

Reply:
This should cover all current `INFORMATION_SCHEMA` references I see in bigquery-etl, though maybe expanding this to also cover any future EU multi-region references would be good, and since we're looking at the original dataset (not the possibly fudged table/view name) it seems like exact matches would be fine.
```suggestion
            dataset_suffix
            and original_dataset not in ("INFORMATION_SCHEMA", "region-eu", "region-us")
        ):
```